src/index.html

```diff
@@ -1,17 +1,118 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Form</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"> 
+    <fieldset class="section">
+      <legend>Personal information</legend>
+      <div class="form">
```

Comment:
Wrap each input in label instead div.
By defoult label has display: inline,
so change it in css on dilsplay: inline-block

Reply:
It can be reason, that some tests failed

Reply:
@VitaliyBondarenko1982 I changed div to label, but I used display: block (inline-block moves all my fieldsets into lines)
now I have the same errors as before
can I ask for your advice?